packages/fxa-payments-server/src/components/SubscriptionTitle/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+import './index.scss';
+
+type SubscriptionTitleProps = {
+  screenType: string;
+};
+
+export const SubscriptionTitle = ({ screenType }: SubscriptionTitleProps) => {
+  const defaultHeaders: { [index: string]: string } = {
+    create: 'Set up your subscription',
+    success: 'Subscription confirmation',
+    processing: 'Confirming subscription...',
+  };
+  return (
+    <div className="subscription-title" data-testid="subscription-title">
+      <Localized id={`subscription-${screenType}-title`}>
+        <h3 className="title">{defaultHeaders[screenType]}</h3>
```

Comment:
The sum type I mentioned above would help here.  `screenType` is used without anything checking that it is one of the keys of `defaultHeaders`.